internal/cmd/tui.go

```diff
@@ -133,7 +133,14 @@ func tuiCmd() *cobra.Command {
 						return err
 					}
 
-					err = runnerClient.RunBlock(ctx, result.block)
+					tempBlock := result.block.Block.Clone()
```

Comment:
```suggestion
					runBlock := result.block.Block.Clone()
```

Minor nitpick, but I personally prefer the name "runBlock" here